rpc/wrtc_client.go

```diff
@@ -152,6 +155,27 @@ func dialWebRTC(
 		}
 	}()
 
+	var (
+		statsMu                                        sync.Mutex
+		callUpdates                                    int
+		maxCallUpdateDuration, totalCallUpdateDuration time.Duration
+	)
+	onICEConnected := func() {
+		// Delay by up to 5s to allow more caller updates/better stats.
+		timer := time.NewTimer(5 * time.Second)
+		select {
+		case <-timer.C:
```

Comment:
q: we only need to delay once right? 

```suggestion
		select {
		case <-time.After(5 * time.Second):
```